website/docs/r/clusterDriver.html.markdown

```diff
@@ -3,17 +3,17 @@ layout: "rancher2"
 page_title: "Rancher2: rancher2_cluster_driver"
 sidebar_current: "docs-rancher2-resource-cluster_driver"
 description: |-
-  Provides a Rancher v2 Cluster Driver resource. This can be used to create Cluster Driver for rancher v2 rke clusters and retrieve their information.
+  Provides a Rancher v2 Cluster Driver resource. This can be used to create Cluster Driver for Rancher v2 RKE clusters and retrieve their information.
```

Comment:
It manages cluster drivers for Rancher 2 Kotainer Engine

```
- Provides a Rancher v2 Cluster Driver resource. This can be used to create Cluster Driver for Rancher v2 RKE clusters and retrieve their information.
+ Provides a Rancher v2 Cluster Driver resource. This can be used to create Cluster Driver for Rancher v2 Kontainer Engine and retrieve their information.
```

Reply:
Is Kontainer Engine a 'publicly' exposed thing? Or is it exposed as RKE?

Reply:
fixed